src/main/java/de/muenchen/isi/infrastructure/entity/stammdaten/SobonJahr.java

```diff
@@ -0,0 +1,30 @@
+package de.muenchen.isi.infrastructure.entity.stammdaten;
+
+import de.muenchen.isi.infrastructure.entity.BaseEntity;
+import java.time.LocalDate;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Entity
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class SobonJahr extends BaseEntity {
```

Comment:
Weshalb wurden in diesem Branch die Stammdaten für die Sobon-Orientierungswerte implementiert? Dieser Branch umfasst die Task ISI-927 zur Umlegung der Fördermixe.